juniper_official/Solutions/Icmp-outlier/icmp.rule

```diff
@@ -103,15 +103,24 @@
             trigger packet-loss {
                 synopsis "ICMP packet loss KPI";
                 description "Sets health based on the packet loss";
-                frequency 60s;
+                /*
+                 * Trigger evaluation time is in offset, The offset value is a 
+                 * multiple of the configured sensor frequency or a configured
+                 * sensor profile.
+                 * e.g. if sensor frequency is 1m and offset is set to 2 then 
+                 * trigger frequency is 2 minutes.
+                 */
+                frequency 1offset;
                 /*
                  * Sets color to red and sends out an anomaly notification when
-                 * the packet ($packet-loss-percent) is 100.
+                 * the packet ($packet-loss-percent) is 100 continuously for
+                 * the offset duration. The offset value is a multiple of the
+                 * configured sensor frequency or a configured sensor profile
```

Comment:
Can you please include a full stop here?

Reply:
Added pullstop

Reply:
Please check now